src/game-logic/reducers/addItemToInventory.test.js

```diff
@@ -0,0 +1,107 @@
+import { addItemToInventory } from './addItemToInventory'
```

Comment:
continuing the default export thread, these would end up not needing the destructure syntax. a minor difference, but destructuring makes me think there are other exports there as well, and even if there were i think it's still nice to have a default one available

Reply:
The import destructuring is an unfortunate bit of syntactical overhead with named exports, for sure. Personally, I think it's balanced out by never needing to think about whether an import is default or named, as you can always assume the latter.

Reply:
yeah that's true. my comment here was moot after your first response anyways. i'm gonna resolve these